File: src/main/source/org/glavo/viewer/util/ImageUtils.java
package org.glavo.viewer.util;

import javafx.scene.image.Image;
import org.jetbrains.annotations.NotNull;

public final class ImageUtils {
    private ImageUtils() {
    }

    @NotNull
    public static Image loadImage(@NotNull String url) {
        return new Image(ImageUtils.class.getResource(url).toExternalForm());
    }
}

File: src/main/java/org/glavo/viewer/util/ImageUtils.kt
@file:JvmName("ImageUtils")
package org.glavo.viewer.util

import javafx.scene.image.Image
import kotlinfx.scene.image
import org.glavo.viewer.gui.Viewer


fun loadImage(url: String): Image = image(Viewer::class.java.getResource(url).toExternalForm())